assets/xml/objects/object_link_child.xml

```diff
@@ -0,0 +1,227 @@
+        <Texture Name="gLinkChildGoronMaskEyeTex" OutName="link_child_goron_mask_eye" Format="rgb5a1" Width="32" Height="32" Offset="0x26120"/>
+        <Texture Name="gLinkChildGoronMaskNoseTex" OutName="link_child_goron_mask_nose" Format="rgb5a1" Width="8" Height="8" Offset="0x26920"/>
+        <Texture Name="gLinkChildGoronMaskHairTex" OutName="link_child_goron_mask_hair" Format="rgb5a1" Width="16" Height="16" Offset="0x269A0"/>
+        <Texture Name="gLinkChildGoronMaskEarTex" OutName="link_child_goron_mask_ear" Format="rgb5a1" Width="8" Height="8" Offset="0x26C20"/><!--TODO verify this-->
+
+        <Texture Name="gLinkChildZoraMaskEyeBoarderTex" OutName="link_child_zora_mask_eye_boarder" Format="rgb5a1" Width="8" Height="8" Offset="0x26CA0"/>
+        <Texture Name="gLinkChildZoraMaskEarTex" OutName="link_child_zora_mask_ear" Format="rgb5a1" Width="32" Height="32" Offset="0x26D20"/>
+        <Texture Name="gLinkChildZoraMaskEyeTex" OutName="link_child_zora_mask_eye" Format="rgb5a1" Width="32" Height="32" Offset="0x275A0"/>
+        <Texture Name="gLinkChildZoraMaskMouthTex" OutName="link_child_zora_mask_mouth" Format="rgb5a1" Width="32" Height="32" Offset="0x27DA0"/>
+
+        <Texture Name="gLinkChildGerudoMaskEyeTex" OutName="link_child_gerudo_mask_eye" Format="rgb5a1" Width="32" Height="32" Offset="0x285A0"/>
+        <Texture Name="gLinkChildGerudoMaskMouthTex" OutName="link_child_gerudo_mask_mouth" Format="rgb5a1" Width="16" Height="16" Offset="0x28DA0"/>
+        <Texture Name="gLinkChildGerudoMaskHairTex" OutName="link_child_gerudo_mask_hair" Format="rgb5a1" Width="16" Height="16" Offset="0x28FA0"/>
+        <Texture Name="gLinkChildGerudoMaskNoseTex" OutName="link_child_gerudo_mask_nose" Format="rgb5a1" Width="8" Height="8" Offset="0x291A0"/>
+
+        <Texture Name="gLinkChildBunnyHoodEyeTex" OutName="link_child_bunny_hood_eye" Format="rgb5a1" Width="16" Height="16" Offset="0x2BA28"/>
+        <Texture Name="gLinkChildBunnyHoodTex" OutName="link_child_bunny_hood" Format="rgb5a1" Width="16" Height="32" Offset="0x2BC28"/>
+        <Texture Name="gLinkChildBunnyHoodEarTex" OutName="link_child_bunny_hood_ear" Format="rgb5a1" Width="16" Height="32" Offset="0x2C028"/>
+        
+        <!--Palettes-->
+        <Texture Name="gLinkChildSkinPAL" OutName="link_child_skin_palette" Format="rgb5a1" Width="16" Height="16" Offset="0x5500"/>
+        <Texture Name="gLinkChildHandPAL" OutName="link_child_hand_palette" Format="rgb5a1" Width="17" Height="4" Offset="0x9E88"/>
+        <Texture Name="gLinkChildSwordsPAL" OutName="link_child_swords_palette" Format="rgb5a1" Width="20" Height="13" Offset="0x9F10"/> <!--For both the kokiri sword sheath and master sword-->
+        <Texture Name="gLinkChildSwordPAL" OutName="link_child_sword_palette" Format="rgb5a1" Width="27" Height="4" Offset="0xA118"/>
+        <Texture Name="gLinkChildBeltPAL" OutName="link_child_belt_palette" Format="rgb5a1" Width="16" Height="16" Offset="0x5300"/>
```

Comment:
TLUT should be used instead of PAL for all these. Also ideally the filenames should be updated to use _tlut instead of _palette, but I think that's relevant for most files in the PR.